Assets/Scripts/Dialogue System/Dialogue Editor/DialogueGraphView.cs

```diff
@@ -0,0 +1,167 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using UnityEditor;
+using UnityEditor.Experimental.GraphView;
+using UnityEditor.UIElements;
+using UnityEngine;
+using UnityEngine.UIElements;
+
+
+public class DialogueGraphView : GraphView
+{
+    public readonly Vector2 defaultNodeSize = new Vector2(150,200);
```

Comment:
This value does not seem to actually be respected, changing it does nothing to the node sizes

Reply:
not changing now